processor/processor.go

```diff
@@ -159,6 +171,13 @@ type Handle struct {
 		eventAuditEnabled               map[string]bool
 		credentialsMap                  map[string][]transformer.Credential
 		nonEventStreamSources           map[string]bool
+		enableWarehouseTransformations  config.ValueLoader[bool]
+	}
+
+	warehouseTransformerStats struct {
```

Comment:
Should `stats` be part of `warehouseTransformation` struct instead of `Handle` struct ?

Reply:
Let me move this to the warehouse->transformer package.